src/main/java/seedu/address/model/candidate/Course.java

```diff
@@ -29,7 +32,17 @@ public Course(String course) {
      * Returns true if a given string is a valid course.
      */
     public static boolean isValidCourse(String test) {
-        return test.matches(VALIDATION_REGEX);
+        if (test.length() > MAX_LENGTH) {
+            return false;
+        }
+
+        for (int index = 0; index < COURSES.length; ++index) {
+            if (test.equals(COURSES[index]) && test.length() == LENGTHS[index]) {
```

Comment:
Perhaps the user input `test` and the courses to match `COURSES[index]` should be lower case? Makes inputting to the `add command` easier as one would not have to capitalize the first letter.

Reply:
@LeeEnEn Yes good point! I thought about this. Let me share with you my thoughts offline.